docs/how-to/prevent-slashing/detect-doppelgangers.md

```diff
@@ -1,32 +1,35 @@
 ---
-title: Enable doppelganger detection
 description: Check if your validators' keys are already active.
-sidebar_position: 7
+sidebar_position: 2
 ---
 
-# Enable doppelganger detection
+# Detect doppelgangers
 
-Doppelganger detection checks if the validators' keys are already active before scheduling any of their duties (the validators stay inactive for at most two epochs). This can help prevent slashing offences.
+Doppelganger detection checks if the validators' keys are already active before scheduling any of their duties (the validators stay inactive for at most two epochs). This can help prevent slashing offenses.
```

Comment:
You could also remove `offences` from `project-words.txt` since this change

Reply:
I updated all the offences to offenses and updated the project-words.txt